docling/datamodel/pipeline_options.py

```diff
@@ -78,9 +134,17 @@ class EasyOcrOptions(OcrOptions):
 
     kind: Literal["easyocr"] = "easyocr"
     lang: List[str] = ["fr", "de", "es", "en"]
-    use_gpu: bool = True  # same default as easyocr.Reader
+    use_gpu: Annotated[
+        int,
```

Comment:
I guess this was a typo, right?

```suggestion
        bool,
```